src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImportPowerShellDataFile.cs

```diff
@@ -0,0 +1,122 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
```

Comment:
Given it is open-source code, I don't think it is needed. @joeyaiello Can you weigh in?